app/Exceptions/User/OldPasswordMismatchException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Exceptions\User;
+
+use Brewmap\Exceptions\ApiException;
+use Illuminate\Http\Response;
+
+class OldPasswordMismatchException extends ApiException
+{
+    protected $message = "Old password is mismatch!";
```

Comment:
There's no need to yell at our users :wink: 

Also `Old password is mismatched.` or `Old password doesn't match.` wouldn't be more grammatically correct?
```suggestion
    protected $message = "Old password doesn't match.";
```